android/src/com/frostwire/android/core/Constants.java

```diff
@@ -180,8 +180,8 @@ private Constants() {
     public static final String SOCIAL_URL_TWITTER_PAGE = "https://twitter.com/frostwire";
     public static final String SOCIAL_URL_REDDIT_PAGE = "https://reddit.com/r/frostwire";
 
-    public static final String VPN_LEARN_MORE_URL = "http://www.frostwire.com/vpn";
+    public static final String VPN_LEARN_MORE_URL = "https://www.expressvpn.com/internet-security/private-vpn/buy?a_aid=frostwire";
```

Comment:
I'm thinking maybe if you want to have a direct link to a partner, and we want to change these URLs, it's better to do so remotely, perhaps this link could be instead something like `http://www.frostwire.com/vpnpartner1`, `http://www.frostwire.com/vpnpartner2`...

and then we point to the desired locations on the server.